python/_tskitmodule.c

```diff
@@ -340,17 +340,25 @@ make_individual_row(const tsk_individual_t *r)
     PyObject *ret = NULL;
     PyObject *metadata = make_metadata(r->metadata, (Py_ssize_t) r->metadata_length);
     PyArrayObject *location = NULL;
-    npy_intp dims;
+    PyArrayObject *parents = NULL;
+    npy_intp dims, nparents;
 
     dims = (npy_intp) r->location_length;
     location = (PyArrayObject *) PyArray_SimpleNew(1, &dims, NPY_FLOAT64);
     if (metadata == NULL || location == NULL) {
         goto out;
     }
     memcpy(PyArray_DATA(location), r->location, r->location_length * sizeof(double));
-    ret = Py_BuildValue("IOO", (unsigned int) r->flags, location, metadata);
+    nparents = (npy_intp) r->parents_length;
```

Comment:
Can reuse ``dims`` here for consistency - slightly confusing for the reader to use ``dims`` for location and ``nparents`` for parents.